src/lib.rs

```diff
@@ -156,6 +156,51 @@ impl Vob<usize> {
         v.mask_last_block();
         v
     }
+
+    /// Create a Vob from a `u8` slice. The most significant bit of each byte comes first in the
+    /// resulting Vob.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #[macro_use] extern crate vob;
+    /// use vob::Vob;
+    /// fn main() {
+    ///     let v = Vob::from_bytes(&[0b10100000, 0b00010010]);
+    ///     assert_eq!(v, vob![true, false, true, false, false, false, false, false,
+    ///                        false, false, false, true, false, false, true, false]);
+    /// }
+    /// ```
+    pub fn from_bytes(slice: &[u8]) -> Vob<usize> {
+        let mut v = Vob::with_capacity(slice.len()
+                                            .checked_mul(8)
+                                            .expect("Overflow detected"));
+        for i in 0..blocks_required::<usize>(slice.len() * 8) {
+            let mut w = usize::zero(); // word
```

Comment:
Without sounding too pedantic, what value does the comment here add? I'd say it could even confusing, since a word is 16-bits on x86_64.

Reply:
"Word" means "native integer size" pretty much everywhere except (maybe only in the past?) Windows AFAIK.

Reply:
> "Word" means "native integer size"

Ah, if only it were that simple :) Brace yourself for pedantry!

I refer you to The "Intel® 64 and IA-32 Architectures Software Developer’s Manual
Volume 1:Basic Architecture":

![x86_64](https://user-images.githubusercontent.com/604955/39763716-cfb50ffe-52d5-11e8-9549-c2170a0ab8b8.png)

Just kill the comment. We all know that a `usize` is the size of a pointer, whereas "word" is not all that portable, sadly.

[Sorry to throw the book at you, especially since in this instance it's such a heavy book! In fact, the book comes in separate volumes, so I'd even have the opportunity to reload!]

Reply:
So on x86_64, a `usize` is a quadword!

Reply:
Intel can define "the" to mean "spade", but I'll still call "the" "the". Wikipedia uses the same definition as I do ;) Anyway, I've killed the comment, but I now wonder how many conversations you and I have had where I've used "word" (==native integer size) and you've heard "16 bits"!

Reply:
I know what you mean, but more eyes than just ours will see this code. It's best to have precise comments, or let the code do the talking.

You've opted for the latter here, so I'm OK with that.